elektronLogging/loggerConfig.go

```diff
@@ -0,0 +1,68 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	elekEnv "github.com/spdfg/elektron/environment"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+type LoggerConfig struct {
+	SchedTraceConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
```

Comment:
Are column headers being added if enabled?

Reply:
That was for future purpose. But I have removed it now